test/utils/link_creator.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
Make this file executable, otherwise it fails if run with anything else other than bash